README.md

```diff
@@ -1,119 +1,103 @@
-
-# Helpful Debugging Commands:
-
-- To run Rails Console:
+*This repo is designed for candidates currently in the interview process with Home Chef. If you stumbled across this guide and think interviewing with us might be fun, please reach out to `hr@homechef.com`.*
+
+---
+
+# The Technical Interview Process
+
+Our goal is to ensure you have a great candidate experience while moving through our interview process. On average, our process takes two weeks to complete. Occasionally steps are adjusted/added/removed, but typically the interview is as follows:
+
+ 1. Phone call with recruiter or engineering manager - .5 hours
+ 2. Behavioral interview with 2 Home Chef engineers - 1.5 hours
+ 3. Technical interview pairing with 2 Home Chef engineers on a real-life ticket - 2 hours
+ 4. Hiring decision is made within ~72 hours of the technical interview
+
+If you are not sure where you are in this process or it has been an extended period of time since you last heard from us, please reach out to `hr@homechef.com`
+
+# Our Interview Philosophy
+
+During the technical interview, our goal is to see what it is like to work with you. You may not complete the exercise in full, and that’s perfectly fine. We have specifically designed our interview to better understand how you think and what it would be like to work with you &mdash; in something close to the environment that we would share if you were hired.
+
+Our technical interview involves pairing with two of our engineers and working through a ticket we've completed recently. Our goal is to give you enough time so you are able to ask questions, get the necessary context needed, and make progress on the problem, ideally without feeling like you are racing against the clock.
+
+# Technical Interview Prep Guide
+
+At Home Chef our tech stack is a mix of React, Redux, HTML, CSS (Tailwind), HAML/Rails views, and JavaScript on the front end. We use a Ruby on Rails back end with a PostgreSQL database. We don't expect you to be an expert in these things.
+
+Our technical interview is designed so you can make progress on it regardless of your specific technical background or level of experience. That being said, familiarity with the concepts and tools below may be helpful during the technical interview:
+  - Basic CRUD app functionality
+  - Interacting with APIs (making GET/POST requests)
+  - React and React Hooks, including state management
+
+# Interview Setup Guide
+### Mac Setup
+
+#### To complete before the interview:
+1. Install [VS code](https://code.visualstudio.com/Download).
+2. Add/install Microsoft's "[Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)" extension.
+
+#### To complete along with your interviewer:
+1. Create ssh config file, `touch ~/.ssh/config`
+    - If you get the error "No such file for directory", create the directory (`mkdir ~/.ssh`) then re-run above command
```

Comment:
Simplified this step a bit, to just always have them touch the file. If it's already there then nothing happens and feels more straight forward to do 1 extra step than navigating diverging instructions

Reply:
That makes sense to me.